wgengine/wgcfg/parser.go

```diff
@@ -79,25 +76,23 @@ func FromUAPI(r io.Reader) (*Config, error) {
 
 	scanner := bufio.NewScanner(r)
 	for scanner.Scan() {
-		line := scanner.Text()
-		if line == "" {
+		line := mem.B(scanner.Bytes())
+		if line.Len() == 0 {
 			continue
 		}
-		parts := strings.Split(line, "=")
-		if len(parts) != 2 {
-			return nil, fmt.Errorf("failed to parse line %q, found %d =-separated parts, want 2", line, len(parts))
+		key, value, ok := memROCut(line, '=')
+		if !ok {
+			return nil, fmt.Errorf("failed to cut line %v on =", line)
```

Comment:
Does mem.RO print nicely? (I don't know offhand.) Do we want `%q`?

Reply:
mem.RO doesn't handle `%q`, but I can convert it into a `string` there cause then we no longer care about perf